tests/Core/_StartCodeCoverage.cmd

```diff
@@ -18,6 +18,8 @@ REM     $:\> explorer C:\Temp\Coverage\index.html
 echo on
 cls
 
+mkdir %USERPROFILE%\AppData\Local\Temp\FluentUI\Coverage
```

Comment:
You don't need to create the folder. This will be done with the `reportgenerator` command.